src/controllers/requestController.js

```diff
@@ -0,0 +1,72 @@
+import model from '../database/models';
+import { findRooms } from '../services/roomService';
+
+// creating a request
+const createRequest = async (req, res) => {
+  try {
+    const { idUser, idRoom, dateStart, dateEnd } = req.body,
+      { id } = req.userData;
+    const existingRoom = await findRooms(idRoom);
+
+    if(!existingRoom) return res.status(400).json({ message: 'Room id does not exist' });
+    console.log(existingRoom);
```

Comment:
Please do not log to console outside your development environment. Check this and all instances of console.log()